templates/shift-scheduling/output.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
The output does not hold the same structure as the other `mip` templates. It would be good to keep the same structure for consistency.

Check out how, for example, the knapsack example looks like:

```json
{
  "options": {
    "limits": {
      "duration": 10000000000
    }
  },
  "solutions": [
    {
      "items": [
        {
          "item_id": "cat",
          "value": 100,
          "weight": 20
        },
        {
          "item_id": "water",
          "value": 40,
          "weight": 2
        },
        {
          "item_id": "phone",
          "value": 6,
          "weight": 1
        },
        {
          "item_id": "book",
          "value": 63,
          "weight": 10
        },
        {
          "item_id": "rx",
          "value": 81,
          "weight": 1
        },
        {
          "item_id": "coat",
          "value": 44,
          "weight": 9
        },
        {
          "item_id": "keys",
          "value": 92,
          "weight": 1
        },
        {
          "item_id": "nuts",
          "value": 18,
          "weight": 4
        }
      ]
    }
  ],
  "statistics": {
    "result": {
      "custom": {
        "constraints": 1,
        "provider": "highs",
        "status": "optimal",
        "variables": 11
      },
      "duration": 0.000540542,
      "value": 444
    },
    "run": {
      "duration": 0.000540542
    },
    "schema": "v1"
  },
  "version": {
    "sdk": "VERSION"
  }
}
```

Also, not sure why we need this file here?